test/backends/xeon/test_launch.py

```diff
@@ -0,0 +1,66 @@
+# Owner(s): ["module: intel"]
+
+from torch.testing._internal.common_utils import TestCase, run_tests, IS_LINUX
+import shutil
+import subprocess
+import tempfile
+import unittest
+
+class TestTorchrun(TestCase):
+    def setUp(self):
+        self._test_dir = tempfile.mkdtemp(prefix=self.__class__.__name__)
+
+    def tearDown(self):
+        shutil.rmtree(self._test_dir)
+
+    @unittest.skipIf(IS_LINUX, "Windows not supported")
```

Comment:
- You can apply this to the class constructor on top if you want to skip all the tests in that class
- You want to skip if it is NOT linux right?
- it is both windows and macos that won't run? The comment could be "Only works on linux" if that's the case.